investment-app/TRINV.Api/TRINV.Domain/Validations/Ensure.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace TRINV.Domain.Validations;
+
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+
+public static class Ensure
+{
+    public static void IsArgumentNullOrEmpty(string? value,
+        [CallerArgumentExpression("value")] string? paramName = null)
+    {
+        if (string.IsNullOrEmpty(value))
+        {
+            throw new ArgumentException("The argument value can't be null or empty", paramName);
```

Comment:
Sorry now i notice that the current methods should be boolean. We don't want to interrupt the flow of the request with exceptions. Please make them all bollean. return true and false;

Reply:
Done.